README.md

```diff
@@ -412,6 +413,35 @@ task :foo do
 end
 ```
 
+### Logger
+
+If you want to convert your log messages to Airbrake errors, you can use our
+integration with Ruby's `Logger` class from stdlib. All you need to do is to
+make sure that the integration is required (if the Airbrake is loaded after
+`Logger`, then it is already integrated):
+
+```ruby
+require 'airbrake/logger/logger_ext'
+```
+
+To start using the integration create a new Logger and assign an Airbrake
+notifier to it:
+
+```ruby
+logger = Logger.new(STDOUT)
+
+# New method.
+# We assign the default notifier here.
+logger.airbrake = Airbrake[:default]
```

Comment:
I had tried this when I implemented it first. This will lead to a lot of spam and unpredicted results. Many libraries use Logger including airbrake-ruby. If we do what you suggest, then Airbrake is getting bombarded by requests, including the ones that we emit during `notify` calls and including from all those libraries that use Logger. This is very unpredictable.